.travis.yml

```diff
@@ -8,10 +8,10 @@ matrix:
   fast_finish: true
   include:
       # Minimum supported versions
-    - php: 7.2
```

Comment:
IMHO we can remove travis CI as we have GitHub CI for years.

The project also seems dead https://app.travis-ci.com/DoctrineEncryptBundle/DoctrineEncryptBundle

Reply:
Good catch.

Reply:
Thanks for noticing that. Will get that done.